FinSearchUnified/Tests/BusinessLogic/Models/FindologicArticleModelTest.php

```diff
@@ -589,4 +590,206 @@ public function testArticleFreeTextFieldAttributes(array $articleConfiguration,
             $this->assertArrayNotHasKey('attr1', $values);
         }
     }
+
+    public function emptyValuesDataProvider()
+    {
+        return [
+            'Summary and Description values are empty' => [
+                [
+                    'name' => 'abdrückklotz-für+butler reifenmontiergerät',
+                    'active' => true,
+                    'tax' => 19,
+                    'supplier' => 'Findologic',
+                    'description' => '',
+                    'descriptionLong' => '',
+                    'categories' => [
+                        ['id' => 3],
+                        ['id' => 5],
+                    ],
+                    'images' => [
+                        ['link' => 'https://via.placeholder.com/300/F00/fff.png'],
+                        ['link' => 'https://via.placeholder.com/300/09f/000.png'],
+                    ],
+                    'mainDetail' => [
+                        'number' => 'FINDOLOGIC2',
+                        'active' => true,
+                        'inStock' => 16,
+                        'prices' => [
+                            [
+                                'customerGroupKey' => 'EK',
+                                'price' => 99.34,
+                            ],
+                        ]
+                    ],
+                ],
+            ],
+        ];
+    }
+
+    /**
+     * @dataProvider emptyValuesDataProvider
+     *
+     * @param array $articleConfiguration
+     *
+     * @throws Exception
+     */
+    public function testEmptyValuesAreNotExported(array $articleConfiguration)
+    {
+        $baseCategory = new Category();
+        $baseCategory->setId(5);
+
+        $articleFromConfiguration = $this->createTestProduct($articleConfiguration);
+
+        $findologicArticle = $this->articleFactory->create(
+            $articleFromConfiguration,
+            'ABCD0815',
+            [],
+            [],
+            $baseCategory
+        );
+
+        $xmlArticle = $findologicArticle->getXmlRepresentation();
+        $actualSummary = $xmlArticle->getSummary();
+        $summary = $actualSummary->getValues();
+        $this->assertEmpty($summary);
```

Comment:
Asserting empty is not sufficient here. Additionally ensure that there are really no elements by checking the count is 0. (`assertCount`). Do this for all assertions that test for empty values.